generators/upgrade/index.js

```diff
@@ -155,7 +155,7 @@ module.exports = class extends BaseGenerator {
 
             assertJHipsterProject() {
                 const done = this.async();
-                if (!this.getJhipsterAppConfig()) {
+                if (!this.getAllJhipsterConfig().baseName) {
```

Comment:
I suggest moving the call to `this.getAllJhipsterConfig()` to a separate variable as the method does some I/O and someone may be tempted to just reuse a similar call as `this.getAllJhipsterConfig().baseName` without assigning the result of `this.getAllJhipsterConfig()` to a variable (and making the I/O calls _n_ times)

Reply:
sure